x-pack/legacy/plugins/siem/public/pages/detection_engine/components/signals_histogram_panel/index.tsx

```diff
@@ -197,46 +228,50 @@ export const SignalsHistogramPanel = memo<SignalsHistogramPanelProps>(
       }
     }, [showLinkToSignals, urlSearch]);
 
+    const titleSize = useMemo(() => (onlyField == null ? 'm' : 's'), [onlyField]);
```

Comment:
I do not think we need to have the `useMemo` here because the component `HeaderSection` should manage primitive props with react.memo